serverless.yml

```diff
@@ -13,33 +13,55 @@ provider:
         - dynamodb:Scan
         - dynamodb:GetItem
         - dynamodb:PutItem
+        - s3:*
       Resource: "*"
   environment:
     PRODUCTS_TABLE: "products"
     STOCKS_TABLE: "stocks"
+    S3_BUCKET: "node-aws-images"
 
 functions:
   getProductsList:
-    handler: products.getProductsList
+    handler: product-service.getProductsList
     events:
       - http:
           path: /products
           method: get
           cors: true
   getProductsById:
-    handler: products.getProductsById
+    handler: product-service.getProductsById
     events:
       - http:
           path: /products/{productId}
           method: get
           cors: true
   createProduct:
-    handler: products.createProduct
+    handler: product-service.createProduct
     events:
       - http:
           path: /products
           method: post
           cors: true
+  importProductsFile:
```

Comment:
According to the task you should have separated serverless.yml files for each service.

Reply:
Noted. I will make these changes in another PR.